tests/func/test_data_cloud.py

```diff
@@ -245,6 +248,10 @@ def test(self):
         for scheme, cl in clist:
             remote_url = scheme + str(uuid.uuid4())
             config[TEST_SECTION][Config.SECTION_REMOTE_URL] = remote_url
+
+            if cl == RemoteHTTP:
```

Comment:
You don't need this.

Reply:
Explained here: https://github.com/iterative/dvc/pull/2411#discussion_r315355621